### PR TITLE
Updated content

### PR DESCRIPTION
**Pull request by keithosayande**
### Description
Updated the content of the 508 bullet point on the Request production access page.

### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [ ] Accessibility concerns have been considered and addressed
  
  - [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

File: src/containers/consumerOnboarding/RequestProductionAccess.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const RequestProductionAccess = (): JSX.Element => (
         API.
       </li>
       <li>
-        All apps and websites must be{' '}
+        We encourage all apps and websites to comply with{' '}
         <a href="https://section508.gov/" target="_blank" rel="noreferrer">
           Section 508
         </a>{' '}
```
Comment (mjlumetta) [inline src/containers/consumerOnboarding/RequestProductionAccess.tsx:32]: gotta delete the "compliant" on the line below, but that's it

**Review by mjlumetta — CHANGES_REQUESTED** [review]
currently reads "We encourage all apps and websites to comply with Section 508 compliant", so need to delete that last word. also, did you want to add the env variable for consumer docs? i added it to my PR, but i imagine yours will get there faster

File: src/containers/consumerOnboarding/RequestProductionAccess.tsx (hunk 1 above)
Comment (keithosayande) [reply src/containers/consumerOnboarding/RequestProductionAccess.tsx:32]: Done.

**Review by mjlumetta — APPROVED** [review]
lgtm ✨ 

do you know why there are only 2 screenshots that got updated?

**Comment by keithosayande** [comment]
> lgtm ✨
> 
> do you know why there are only 2 screenshots that got updated?

This PR on changed 1 thing?

**Comment by mjlumetta** [comment]
the screenshot thing is because the screenshot is getting taken twice right now (old URL + new URL), and each screenshot is taken at 3 viewports (mobile, tablet, and desktop, roughly). so i would expect 6 files to change rather than 2, but there could be some reason for that that i'm not aware of.

**Review by mjlumetta — APPROVED** [review]

**Pull request merged**